packages/cw-derive/tests/messages_generation.rs

```diff
@@ -0,0 +1,36 @@
+use cosmwasm_std::{Addr, Decimal, Response};
+
+pub struct Ctx;
+pub struct Error;
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq, schemars::JsonSchema)]
+pub struct QueryResult;
+
+#[cw_derive::interface]
+pub trait Interface {
+    #[msg(exec)]
+    fn no_args_execution(&self, ctx: Ctx) -> Result<Response, Error>;
+
+    #[msg(exec)]
+    fn argumented_execution(&self, ctx: Ctx, addr: Addr, coef: Decimal, desc: String);
```

Comment:
Shouldn't this have a return value? Shall we/ can we enforce `Result<Response, E>`?

Reply:
It should. Corrected in follow up. I will not add it here to avoid rebasing issues.